tests/validators/tests.py

```diff
@@ -264,6 +264,13 @@
     (DecimalValidator(2, 1), Decimal('0.99'), ValidationError),
     (DecimalValidator(3, 1), Decimal('999'), ValidationError),
     (DecimalValidator(4, 1), Decimal('999'), None),
+    (DecimalValidator(20, 2), Decimal('742403889818000000'), None),
+    (DecimalValidator(20, 2), Decimal('7.42403889818E+17'), None),
```

Comment:
This looks like the only test failure that's fixed so the other tests should go in the first commit of "existing passing tests". Similarly, the commit message could be improved to describe more specifically what situation is fixed considering some scientific notation seems to already work.